Windows github workflow is broken

```
  --- stderr
  gmake: *** No targets specified and no makefile found.  Stop.
  touch: cannot touch `D:\\..\\.cargo\\git\\checkouts\\mozjs-fa11ffc7d4f1cc2d\\c6c7b53\\mozjs/js/src/configure': No such file or directory
  touch: cannot touch `D:\\..\\.cargo\\git\\checkouts\\mozjs-fa11ffc7d4f1cc2d\\c6c7b53\\mozjs/js/src/old-configure': No such file or directory
  mozmake[1]: Makefile: No such file or directory
  mozmake[1]: *** No rule to make target 'Makefile'.  Stop.
  mozmake: *** [D:\..\.cargo\git\checkouts\mozjs-fa11ffc7d4f1cc2d\c6c7b53\makefile.cargo;183: all] Error 2
  thread 'main' panicked at 'assertion failed: result.success()', D:\..\.cargo\git\checkouts\mozjs-fa11ffc7d4f1cc2d\c6c7b53\build.rs:177:5
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Error: Process completed with exit code 101.
```

--- a/.github/workflows/rust.yml
+++ b/.github/workflows/rust.yml
@@ -67,6 +67,6 @@ jobs:
         NATIVE_WIN32_PYTHON: "C:\\mozilla-build\\python\\python.exe"
         PYTHON3: "C:\\mozilla-build\\python3\\python3.exe"
         LIBCLANG_PATH: "C:\\ProgramData\\scoop\\apps\\llvm\\current\\lib"
-        CARGO_HOME: ${{ env.GITHUB_WORKSPACE }}\..\.cargo
+        CARGO_HOME: ${{ github.workspace }}\..\.cargo
       run: |
         cargo test --verbose --verbose ${{ matrix.features }} --lib

